libs/settings/public/feature-settings-modal/src/lib/settings-modal.component.html

```diff
@@ -21,4 +21,7 @@ <h2>Dashboard Settings</h2>
   <div>
     {{ history$ | async | json }}
   </div>
+  <div class="menu-items-container" *ngFor="let menuItem of menuItems$ | async">
+    <architect-poc-menu-item [menuItem]="menuItem"></architect-poc-menu-item>
```

Comment:
one last tiny change request. could u change the selector to `ds` for design system? you should adjust the `eslint` config for this lib as well